server/routes/findInterventions/InterventionDetailsPresenter.ts

```diff
@@ -0,0 +1,82 @@
+import { Eligibility, Intervention } from '../../services/interventionsService'
+import { SummaryListItem } from '../../utils/summaryList'
+import utils from '../../utils/utils'
+
+export interface InterventionDetails {
+  title: string
+  href: string
+  body: string
+  summary: SummaryListItem[]
+}
+
+export default class InterventionDetailsPresenter {
+  constructor(private readonly intervention: Intervention) {}
+
+  get title(): string {
+    return this.intervention.title
+  }
+
+  get href(): string {
+    return `/find-interventions/intervention/${this.intervention.id}`
+  }
+
+  get body(): string {
+    return this.intervention.description
+  }
+
+  get summary(): SummaryListItem[] {
+    return [
+      {
+        key: 'Type',
+        lines: ['Dynamic Framework'],
+        isList: false,
+      },
+      {
+        key: 'Location',
+        lines: [this.intervention.pccRegions.map(region => region.name).join(', ')],
+        isList: false,
+      },
+      {
+        key: 'Criminogenic needs',
+        lines: [utils.convertToProperCase(this.intervention.serviceCategory.name)],
+        isList: false,
+      },
+      {
+        key: 'Provider',
+        lines: [this.intervention.serviceProvider.name],
+        isList: false,
+      },
+      {
+        key: 'Age group',
+        lines: [this.ageGroupDescription(this.intervention.eligibility)],
+        isList: false,
+      },
+      {
+        key: 'Sex',
+        lines: [this.sexDescription(this.intervention.eligibility)],
+        isList: false,
+      },
+    ]
+  }
+
+  private ageGroupDescription(eligibility: Eligibility): string {
```

Comment:
Is there a reason you changed these from `static` methods? I usually use `static` to make it clear at a glance that the behaviour / return value of a method isn't affected by any instance state.

Reply:
Yeah, something about calling static methods inside instance methods felt a bit off to me, I'm not sure why, as we call stuff like `Thing.new` all the time in Ruby. I'll change this.